public/js/test/mochitest/browser_dbg-breaking.js

```diff
@@ -0,0 +1,41 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Tests the breakpoints are hit in various situations.
```

Comment:
I was planning on doing a bfcache test as a separate one, because I'll have to do some special navigation commands to make sure it loads from the bfcache. But I'll think about it, maybe I will go ahead and do that.